docs/design/layout.md

```diff
@@ -369,4 +369,83 @@ So even though a `measureSpec` is an integer, you cannot simply add or subtract
+
+This brings us to the second main difference: the MAUI VerticalStackLayout and HorizontalStackLayout do not recognize the "AndExpand" layout options. If they see a child with layout options that include "AndExpand", they simply treat it as if the "AndExpand" weren't there - e.g., `FillAndExpand` becomes `Fill`. 
+
+For simplicity of migration from Forms to MAUI, the MAUI.Controls StackLayout _does_ honor "AndExpand", at least for the time being. All of the "AndExpand" options have been marked `Obsolete`. If you want to avoid the warning about the obsolete properties, you should convert your layouts which use "AndExpand" options to the appropriate layout type. The suggested process is as follows:
+
+1. If your layout is anything other than a StackLayout, remove all uses of "AndExpand". Just as in Xamarin.Forms, the "AndExpand" options have no effect on any layout other than StackLayout. If your layout wasn't a StackLayout, "AndExpand" was never doing anything. 
+
+2. Remove any "AndExpand" properties which are orthogonal to the stacking direction. For example, if you have a StackLayout with an Orientation of `Vertical`, and it has a child with a `HorizontalAligment="CenterAndExpand"` - that "AndExpand" does nothing. You can just remove it. 
+
+3. If you have any remaining "AndExpand" properties on a StackLayout, you should convert that StackLayout to a Grid; the Grid is designed to subdivide a space, and will provide the layout that "AndExpand" provided in Xamarin.Forms. For example,
+
+```
+<StackLayout>
+	<Label Text="howdy"/>
+	<Image VerticalOptions="FillAndExpand" src="dotnetbot.png"/>
+</StackLayout>
+```
+
+can be converted to 
+
+```
+<Grid RowDefinitions="Auto, *">
+	<Label Text="howdy"/>
+	<Image Grid.Row="1" src="dotnetbot.png"/>
+</StackLayout>
+```
+ 
+Anything that was marked "AndExpand" should go in its own row or column with a size of "*".
+
+### ScrollView Changes
+
+The other main difference is that the Xamarin.Forms ScrollView does not behave consistently when stacking. It has some arbitary limits on minimum size which depend partially on its content, and it will compress to allow other items to fit on the screen inside a StackLayout in ways that are inconsistent and sometimes surprising. 
+
+MAUI, on the other hand, simply allows the ScrollView to expand to whatever size it wants to be unless otherwise constrained. This means that inside of a VerticalStackLayout, which can expand infinitely, a ScrollView will simply expand to its full content height and does not scroll. This can be a little surprising for Forms users. Remember, StackLayouts simply continue in their stacking direction until they run out of content; they do not subdivide their container along that axis. If you want to limit your content to a constrained space in a direction, you should use another control, like a Grid.
```

Comment:
"Wants to be" is vague and just as arbitrary. Can this be clearly defined to what constitutes a size target for ScrollView?

Reply:
Updated.